src/components/pages/Trip.js

```diff
@@ -21,14 +21,18 @@ export default function Trip({ trips }) {
     <TripStyled>
       <img src={defaultImg} width="200" alt=""></img>
       <WrapperStyled>
-        <Date>
-          <FontAwesomeIcon className="icon" icon={['fal', 'calendar-alt']} />
-          {trip.date}
-        </Date>
+        {trip.date.length ? (
```

Comment:
better use && instead of the ternary operator See:
https://reactjs.org/docs/conditional-rendering.html (Inline If with Logical && Operator)